components/NavigationBar/index.js

```diff
@@ -5,33 +5,39 @@ import { FaHouse, FaPlus, FaChartPie } from "react-icons/fa6";
 function NavigationBar() {
   const router = useRouter();
 
-  const isActive = (path) => {
-    return router.pathname === path;
-  };
+ function isActive(path) {
+   return router.pathname === path;
+ }
 
-  return (
-    <Styled.NavBar>
-      <Styled.NavContainer>
-        <Styled.NavLink onClick={() => router.push("/")} active={isActive("/")}>
-          <FaHouse />
-        </Styled.NavLink>
+ return (
+   <Styled.NavBar>
+     <Styled.NavContainer>
+       <Styled.NavLink
+         key="home"
+         onClick={() => router.push("/")}
```

Comment:
please use href instead here: 
```suggestion
         href='/'
```